src/lib/components/fileUploader/fileUploader.md

```diff
@@ -0,0 +1,12 @@
+```js
```

Comment:
Is it possible to get a bit more detail in the .md when you get a second Kevin,

Reply:
sure, I have to add more information...